kie-wb-tests/kie-wb-tests-rest/pom.xml

```diff
@@ -1,147 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <parent>
-    <groupId>org.kie</groupId>
-    <artifactId>kie-wb-tests</artifactId>
-    <version>7.6.0-SNAPSHOT</version>
-  </parent>
-  <modelVersion>4.0.0</modelVersion>
-
-  <artifactId>kie-wb-tests-rest</artifactId>
-
-  <name>KIE (Drools) Workbench Tests :: REST API</name>
```

Comment:
Why is this test suite removed? In this test suite were integration tests for KIE-WB REST API which runs on application containers. I went through all PRs for AF-648 and I didn't find these tests elsewhere.